server/src/main/scala/io/delta/sharing/server/config/ServerConfig.scala

```diff
@@ -65,7 +65,7 @@ case class ServerConfig(
       authorization = null,
       ssl = null,
       host = "localhost",
-      port = 80,
+      port = 8080,
```

Comment:
Changing the default value would be a breaking change for people using a config without the port config. It's better to not change this.

Reply:
I see. Do we need to say anything in the config template?

Reply:
nvm, saw the comment above